MediaBrowser.Controller/Session/SessionInfo.cs

```diff
@@ -123,7 +124,6 @@ public string[] PlayableMediaTypes
         /// Gets or sets the session controller.
         /// </summary>
         /// <value>The session controller.</value>
-        [JsonIgnore]
```

Comment:
Why remove this?

Reply:
Does this hide the object from the JSON response? I forgot to remove this change from when I was trying to figure out what it did.

Reply:
It hides it from the System.Text.Json serializer